api_utils/user_agent.py

```
@@ -0,0 +1,18 @@
+"""
+
+from .api_utils.user_agent import default_user_agent
+
+"""
+import os
+
+def default_user_agent():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)